fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -263,23 +313,18 @@ public PlanFragment visitPhysicalHashJoin(
         // NOTICE: We must visit from right to left, to ensure the last fragment is root fragment
         PlanFragment rightFragment = visit(hashJoin.child(1), context);
         PlanFragment leftFragment = visit(hashJoin.child(0), context);
-        PhysicalHashJoin physicalHashJoin = hashJoin.getOperator();
-
-        //        Expression predicateExpr = physicalHashJoin.getCondition().get();
-        //        List<Expression> eqExprList = Utils.getEqConjuncts(hashJoin.child(0).getOutput(),
-        //                hashJoin.child(1).getOutput(), predicateExpr);
-        JoinType joinType = physicalHashJoin.getJoinType();
-
         PlanNode leftFragmentPlanRoot = leftFragment.getPlanRoot();
         PlanNode rightFragmentPlanRoot = rightFragment.getPlanRoot();
+        PhysicalHashJoin physicalHashJoin = hashJoin.getOperator();
+        JoinType joinType = physicalHashJoin.getJoinType();
 
         if (joinType.equals(JoinType.CROSS_JOIN)
```

Comment:
this PR only change translate join from shuffled join to broadcast join. btw, in translator we should translate all situation, should not depend on implementation and exploration job in cascades. because when we test cascades framework, we will forbidden some rules

Reply:
Then if we encounter a `PhysicalHashJoin` whose `JoinType` is cross join, an error should be reported directly here.